WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -748,14 +775,20 @@ private extension CollectOrderPaymentUseCase {
                 if let email = order.billingAddress?.email, email.isNotEmpty {
                     receiptState = .paymentSuccessEmailSent(email: email)
                 } else {
-                    receiptState = .promptToSendEmailReceipt(emailReceiptAction: {
-                        // TODO
+                    receiptState = .promptToSendEmailReceipt(emailReceiptAction: { [weak self] in
+                        guard let self else { return }
+                        presentSendReceiptAfterPayment { order in
```

Comment:
nit: might we want to weak self in this closure as well?
```suggestion
                        presentSendReceiptAfterPayment { [weak self] order in
```